spec/presenters/queries/link_set_presenter_spec.rb

```diff
@@ -2,15 +2,16 @@
 
 RSpec.describe Presenters::Queries::LinkSetPresenter do
   describe ".present" do
+    let(:content_id) { SecureRandom.uuid }
+    let(:link_set) { FactoryGirl.create(:link_set, content_id: content_id) }
+
     before do
       FactoryGirl.create(:lock_version, target: link_set, number: 101)
       @result = Presenters::Queries::LinkSetPresenter.present(link_set)
```

Comment:
I think it'd be more rspec idiomatic to move this to be the subject e.g `subject(:result) { Presenters... }`